src/cluster/src/communication.rs

```diff
@@ -624,8 +639,8 @@ mod turmoil_tests {
             }
 
             sim.step().unwrap();
-            if sim.elapsed() > Duration::from_secs(120) {
-                panic!("simulation not finished after 120s");
+            if sim.elapsed() > Duration::from_secs(60) {
+                panic!("simulation not finished after 60s");
```

Comment:
I checked and the test failure that motivated the 60s->120s bump (https://github.com/MaterializeInc/materialize/pull/32306) was caused by the same bug, so it should be safe to unbump.